modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesPovisioningConstants.java

```diff
@@ -42,6 +42,6 @@
     public static final String RULES = "rules";
     public static final String HOST = "host";
 
-    public static final String VOLUME_MOUNT = "";
-    public static final String VOLUME_MOUNT_PATH = "";
+    public static final String VOLUME_MOUNT = "default";
+    public static final String VOLUME_MOUNT_PATH = "/etc/default";
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/c707570879201754759203ff99b9c5ea5489dcba